File: src/test/java/se/johannesdahlgren/aoc25/Day1Test.java
package se.johannesdahlgren.aoc25;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class Day1Test {

  @Test
  void day1Part1Example() {
    Day1 day1 = new Day1(".example");
    int actualPassword = day1.actualPassword();
    assertThat(actualPassword).isEqualTo(3);
  }

  @Test
  void day1Part1() {
    Day1 day1 = new Day1("");
    int actualPassword = day1.actualPassword();
    assertThat(actualPassword).isEqualTo(1011);
  }
}

File: src/test/kotlin/se/johannesdahlgren/aoc22/Day1Test.kt
package se.johannesdahlgren.aoc22


import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class Day1Test {

    @Test
    fun part1Example() {
        val day1 = Day1("day1Example")
        assertThat(day1.getCarriedCaloriesByElfCarryingMost()).isEqualTo(24000)
    }

    @Test
    fun part1() {
        val day1 = Day1("day1")
        assertThat(day1.getCarriedCaloriesByElfCarryingMost()).isEqualTo(72511)
    }

    @Test
    fun part2Example() {
        val day1 = Day1("day1Example")
        assertThat(day1.getCarriedCaloriesByTop3ElvesCarryingMost()).isEqualTo(45000)
    }

    @Test
    fun part2() {
        val day1 = Day1("day1")
        assertThat(day1.getCarriedCaloriesByTop3ElvesCarryingMost()).isEqualTo(212117)
    }
}